### ISSUE TITLE
HEAD moved to "Initial commit"

### ISSUE THREAD

**Issue by wrzaskun** [coders-school/git-checker#65]
In My opinion this is low level priority error, because this is only other idea to solve the excercise.
`HEAD moved to Initial commit` this error occurs when branch is created by
`git checkout -b cs HEAD^` instead of 
`git checkout -b cs` then `git reset --hard HEAD^` or similar. 
This is only shortcut for replace two line of code to one.

**Comment by ziobron** [comment]
🏅 2 XP granted for @wrzaskun
Thanks for raising this Issue to help to make the course better!

**Comment by ziobron** [comment]
That's true. Thanks for a good suggestion. Would you try to open a PR with the suggested change? :)
